### PR TITLE
Mentions & replies AC notifications

### LINKED ISSUE #7275: Mention notifications
Epic:  https://github.com/status-im/status-desktop/issues/7070

### Description

- [x] Mention in a community
- [x] Mention in a group chat

### PR DESCRIPTION
**Pull request by MishkaRogachev**
Close #7275

### What does the PR do

Making mention AC notification match design and specs

### Affected areas

Activity Center

### Screenshot of functionality (including design for comparison)

Screenshot 2022-09-23 at 17 15 48

### CODE REVIEW

File: ui/app/mainui/activitycenter/views/ActivityNotificationMention.qml [hunk 1]
```diff
@@ -9,7 +9,47 @@ import shared 1.0
...
+
+    Component {
+        id: communityBadgeComponent
+
+        CommunityBadge {
+            id: communityBadge
+
+            property var community: root.store.getCommunityDetailsAsJson(notification.message.communityId)
+            // TODO: here i need chanel
+            // property var channel: root.store.chatSectionModule.getItemAsJson(notification.chatId)
```
Comment (MishkaRogachev) [inline ui/app/mainui/activitycenter/views/ActivityNotificationMention.qml:27]: Help wanted! Is there an API to fetch channel name and color using chatId?

**Comment by caybro** [comment]
Needs a proper rebase

File: ui/app/mainui/activitycenter/views/ActivityNotificationMention.qml (hunk 1 above)
Comment (MishkaRogachev) [reply ui/app/mainui/activitycenter/views/ActivityNotificationMention.qml:27]: This thing is quite blocking, created new issue for that: https://github.com/status-im/status-desktop/issues/7529

File: ui/app/mainui/activitycenter/controls/Badge.qml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import QtQuick 2.14
+import QtQuick.Layouts 1.14
```
Comment (caybro) [inline ui/app/mainui/activitycenter/controls/Badge.qml:2]: Layouts needed? ;)

File: ui/app/AppLayouts/Chat/stores/RootStore.qml [hunk 3]
```diff
@@ -371,7 +371,19 @@ QtObject {
...
+    }
+
+    function getCommunityDetailsAsJson(id) {
+        let jsonObj = communitiesModuleInst.getCommunityDetails(id)
+        try {
+            let obj = JSON.parse(jsonObj)
+            return obj
+        }
+        catch (e) {
+            console.debug("error parsing community by id: ", id, " error: ", e.message)
```
Comment (caybro) [inline ui/app/AppLayouts/Chat/stores/RootStore.qml:384]: `console.warn()` even

**Review by caybro — APPROVED** [review]
Looks good from my POV

File: ui/app/mainui/AppMain.qml [hunk 4]
```diff
@@ -39,14 +39,16 @@ import AppLayouts.stores 1.0
...
 
 Item {
     id: appMain
     anchors.fill: parent
 
     property alias appLayout: appLayout
     property alias dragAndDrop: dragTarget
-    property RootStore rootStore: RootStore { }
+    property RootStore rootStore: RootStore {}
+    property AC.RootStore acStore: AC.RootStore {}
```
Comment (MishkaRogachev) [inline ui/app/mainui/AppMain.qml:51]: @alexandraB99 can you pls advice me a good place for instancing AC root store?

**Review by iurimatias — APPROVED** [review]

**Pull request merged**